test_launch_ros/test/test_launch_ros/actions/test_lifecycle_node.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the LifecycleNode Action."""
+
+from launch import LaunchContext
+from launch_ros.actions import LifecycleNode
+
+import pytest
+
+
+def test_lifecycle_node_constructor():
+    # Construction without name
+    with pytest.raises(TypeError):
+        LifecycleNode(
+            package='asd',
+            executable='bsd',
+            name='my_node',
+        )
+    # Construction without namespace
```

Comment:
@ivanpauno nit: comments do not match assertions

Reply:
Ups, fixed in https://github.com/ros2/launch_ros/pull/157/commits/620801e6efd80713b5a19eaa98e17ae630bfbb1c.